pom.xml

```diff
@@ -9,7 +9,8 @@
     <version>1.0-SNAPSHOT</version>
 
     <properties>
-        <ignite.version>2.14.0</ignite.version>
+	    <ignite.version>2.15.0</ignite.version>
```

Comment:
There needs to be a matching Control Center Agent. The 2.14 agent with AI2.15 isn't supported, though they concede that it'll probably work.

Let's wait until the new agent is released -- they're saying next week -- before bumping to the current Ignite version.